### ISSUE TITLE
Is it possible to control the `str` display of ModelMultipleChoiceFilter results?

### ISSUE THREAD

**Issue by shacker** [carltongibson/django-filter#1119]
When using ModelMultipleChoiceFilter, the returned display value of each object in the returned queryset is the `str()` method of the model being queried. In most cases you can control this by altering the `str()` method of the queried model, but in our case we don't want to - we want to control it at the ModelMultipleChoiceFilter level.

Is this currently possible? If not, this is a feature request :)

**Comment by rpkilby** [comment]
Filters use Django form fields under the hood, so this should be more a general question of "how do I modify the choice labels for a `ModelMultipleChoiceField`?"

You should be able to do this by implementing [`YourModel.label_from_instance`](https://docs.djangoproject.com/en/2.2/ref/forms/fields/#modelmultiplechoicefield).

**Comment by shacker** [comment]
Aha! Excellent information, thanks. This might be a useful mention in the docs (though I understand it doesn't make sense to mention it in many places). Thanks!

**Comment by rpkilby** [comment]
Yeah - one thing I'd like to clarify the docs is the relationship between filterset/form and filter/field.
